clients/client-python/gravitino/catalog/base_schema_catalog.py

```diff
@@ -78,7 +78,7 @@ def list_schemas(self, namespace: Namespace) -> [NameIdentifier]:
             resp.body, infer_missing=True
         )
         entity_list_response.validate()
-        return entity_list_response.identifiers()
+        return [str(identifier) for identifier in entity_list_response.identifiers()]
 
     def create_schema(
```

Comment:
The "createSchema", "loadSchema", "alterSchema" and "dropSchema" all need be changed to use a string parameter to replace the NameIdentifier parameter.

Reply:
done!